tests/functional/adapter/test_constraints.py

```diff
@@ -23,8 +23,29 @@
 """
 
 class TestRedshiftConstraintsColumnsEqual(BaseConstraintsColumnsEqual):
-    pass
+    @pytest.fixture
+    def int_array_type(self):
+        return "INTEGER_ARRAY"
 
+    @pytest.fixture
+    def string_array_type(self):
```

Comment:
this fixture and the `int_array_type` one shouldn't be necessary anymore! `BaseConstraintsColumnsEqual` doesn't do anything with them.